### PR TITLE
release: 0.3.6

### PR DESCRIPTION
**Pull request by jjfrench**
## Summary:
- bumping versions and adding release to HISTORY for release

## Checklist before requesting a review:
- [ ] My code follows the guidelines of this project
- [x] I  performed a self-review of my code changes
- [x] I have added my changes to the change log (`HISTORY.rst`)
- [ ] I have completed spell-checking, removed unnecessary print statements & commented-out code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -45,7 +45,7 @@ History
 
 * Bug fix, error in manifest of files to include in python package (#120)
 
-Unreleased
```
Comment (wildintellect) [inline HISTORY.rst:48]: Should the Unreleased heading remain, so it doesn't have to be remembered on the next edit?

File: HISTORY.rst [hunk 2]
```diff
@@ -45,7 +45,7 @@ History
...
+0.3.6 (2023-11-29)
 ----------------------
 
 * Updated datasets for MAAP STAC
```
Comment (wildintellect) [inline HISTORY.rst:51]: Hmm, didn't we remove Stamen base layers? This also impacts VEDA, and possibly all users if it's hard coded into the library.

Comment (jjfrench) [reply HISTORY.rst:51]: Temporarily disabled - just commented the layer out. The Stamen post said that they were migrating the maps to be hosted somewhere, so they might become available through ipyleaflet again. I'll update the log

Comment (wildintellect) [reply HISTORY.rst:51]: https://docs.stadiamaps.com/guides/migrating-from-stamen-map-tiles/ it moved to Stadia which might incur costs to users and would require more work to implement a token authenticated layer.

My point was the Changelog should say remove/disabled Stamen layers.

**Review by emmalu — APPROVED** [review]

**Review by wildintellect — APPROVED** [review]

**Pull request merged**